### PR TITLE
Added builder rank

### PR DESCRIPTION
**Pull request by ThePheonixBlader**
Adds a builder rank (Requested by Aggelos)

### CODE REVIEW

**Comment by DUFCLiam** [comment]
Change it to Aqua the tag and leave builder gold

**Comment by aggelosQQ** [comment]
No, just make it all aqua:P 
If this is correct, please merge it.

**Comment by DUFCLiam** [comment]
Ok.

File: src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
     IMPOSTOR("an " + ChatColor.YELLOW + ChatColor.UNDERLINE + "Impostor", ChatColor.YELLOW.toString() + ChatColor.UNDERLINE + "[IMP]"),
     NON_OP("a " + ChatColor.GREEN + "Non-OP", ChatColor.GREEN.toString()),
     OP("an " + ChatColor.RED + "OP", ChatColor.RED + "[OP]"),
+    BUILDER("a " + ChatColor.GOLD + "Builder!", ChatColor.GOLD + "[Builder]"),
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java]: Trailing `!` remove `!` and change it to AQUA

**Comment by ThePheonixBlader** [comment]
#110 #109 #108 #107

**Comment by ThePheonixBlader** [comment]
#111

**Comment by aggelosQQ** [comment]
I'd say wait for @LegendIsAwesomes or @RobinGall2910 to review this before merging it.

**Pull request merged**